android/capacitor/src/main/java/com/getcapacitor/annotation/Permission.java

```diff
@@ -9,11 +9,11 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Permission {
     /**
-     * The Android string for the permission.
-     * Eg: Manifest.permission.ACCESS_COARSE_LOCATION
-     *     or "android.permission.ACCESS_COARSE_LOCATION"
+     * An array of Android permission strings.
+     * Eg: {Manifest.permission.ACCESS_COARSE_LOCATION}
+     *     or {"android.permission.ACCESS_COARSE_LOCATION"}
      */
-    String permission() default "";
+    String[] permission() default {};
```

Comment:
Thoughts on pluralizing it now that it's an array?

```suggestion
    String[] permissions() default {};
```

Reply:
It makes sense because it can be one or many but I hesitate because the parent element is called "permissions" which is made up of one or many `@Permission` annotations, which then can contain one or many "permission" manifest constants. Im thinking of renaming it to something like "constants" or "strings" but that sounds awfully generic

Reply:
I don't mind `strings`, because that's what they're called in Android and we already have the context of permissions given the `@Permission` annotation. 

```java
@Permission(strings = { Manifest.permission.WRITE_EXTERNAL_STORAGE, Manifest.permission.READ_EXTERNAL_STORAGE }, alias = "publicStorage")
```

^ This looks good to me. 😁